server/src/main/java/mineclone/common/world/flags/SetBlockFlags.java

```diff
@@ -0,0 +1,10 @@
+package mineclone.common.world.flags;
+
+public class SetBlockFlags {
```

Comment:
Originally I moved them out because the `IServerWorld` interface included not only `setBlockState` flags, but also power related flags, and it felt rather cluttered. I think grouping them together in a class makes it a lot cleaner.